runtime/themes/base16_default_dark.toml

```diff
@@ -33,11 +33,12 @@
 "namespace" = "base0E"
 "ui.help" = { fg = "base06", bg = "base01" }
 
-"info" = "base03"
-"hint" = "base03"
-"debug" = "base03"
-"diagnostic" = "base03"
-"error" = "base08"
+"info" = { fg = "base03", bg = "base01" }
+"hint" = { fg = "base03", bg = "base01" }
+"debug" = { fg = "base03", bg = "base01" }
+"diagnostic" = { fg = "base03", bg = "base01" }
```

Comment:
`diagnostic` is the style of the text that has some diagnostic. Other themes set this to `modifiers = ["underline"]` and I don't recommend a `bg`.

Reply:
Changed to `"diagnostic" = { modifiers = ["underlined"] }`